js/private/lifecycle/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"Private lifecycle files"
+
+load("//js:js_binary.bzl", "js_binary")
+load("@aspect_rules_js_lifecycle_npm_deps//:node_modules.bzl", "node_modules")
+load("@aspect_rules_js_lifecycle_npm_deps//:package.bzl", "package")
+
+node_modules()
+
+js_binary(
+    name = "lifecycle-hooks",
+    data = [
+        package("@pnpm/lifecycle"),
```

Comment:
You can make this `@aspect_rules_js_lifecycle_npm_deps//@pnpm/lifecycle` and avoid using the `package` function